contrib/cirrus/runner.sh

```diff
@@ -457,7 +453,9 @@ function _bail_if_test_can_be_skipped() {
     # Defined by Cirrus-CI for all tasks
     # shellcheck disable=SC2154
     head=$CIRRUS_CHANGE_IN_REPO
-    base=$(git merge-base $DEST_BRANCH $head)
+    # shellcheck disable=SC2154
+    base=$PR_BASE_SHA
+    echo "_bail_if_test_can_be_skipped: head=$head  base=$base"
```

Comment:
debugging printf. Does anyone mind if I leave it in for a little while?

Reply:
It's fine IMO